projects/angular-cld/src/lib/cloudinary-sdk-analytics.spec.ts

```diff
@@ -0,0 +1,217 @@
+import { Component, DebugElement } from '@angular/core';
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Cloudinary } from './cloudinary.service';
+import CloudinaryConfiguration from './cloudinary-configuration.class';
+import { CloudinaryImage } from './cloudinary-image.component';
+import { CloudinaryTransformationDirective } from './cloudinary-transformation.directive';
+import {LazyLoadDirective } from './cloudinary-lazy-load.directive';
+import { CloudinaryPlaceHolder } from'./cloudinary-placeholder.component';
```

Comment:
@strausr , this PR, besides adding analytics, also solves an issue with sending config items to the placeholder (like secure, etc.)
Did you add tests for those use-cases?

Reply:
There is a ticket to add tests for it this sprint